data-es6.js

```diff
@@ -10540,12 +10543,12 @@ exports.tests = [
         set.add(123);
         set.add(123);
 
-        return set.has(123);
+        return set instanceof S && set.has(123);
```

Comment:
Ah, I see what you mean - because you're not overwriting `add` or `has`, it's falling back to the `Set.prototype` methods. I still feel it'd be more robust doing an explicit `Set.prototype.has.call` etc, but I withdraw any previously implied objection :-) Thanks for bearing with me and explaining!